pynest/examples/brunel_alpha_nest.py

```diff
@@ -171,25 +168,16 @@ def ComputePSPnorm(tauMem, CMem, tauSyn):
 
 print("Building network")
 
-###############################################################################
-# Configuration of the model ``iaf_psc_alpha`` and ``poisson_generator`` using
-# ``SetDefaults``. This function expects the model to be the inserted as a
-# string and the parameter to be specified in a dictionary. All instances of
-# theses models created after this point will have the properties specified
-# in the dictionary by default.
-
-nest.SetDefaults("iaf_psc_alpha", neuron_params)
-nest.SetDefaults("poisson_generator", {"rate": p_rate})
-
 ###############################################################################
 # Creation of the nodes using ``Create``. We store the returned handles in
 # variables for later reference. Here the excitatory and inhibitory, as well
 # as the poisson generator and two spike detectors. The spike detectors will
-# later be used to record excitatory and inhibitory spikes.
+# later be used to record excitatory and inhibitory spikes. Properties of the
+# nodes are specified via ``param``, which expects a dictionary.
 
-nodes_ex = nest.Create("iaf_psc_alpha", NE)
-nodes_in = nest.Create("iaf_psc_alpha", NI)
-noise = nest.Create("poisson_generator")
+nodes_ex = nest.Create("iaf_psc_alpha", NE, param=neuron_params)
```

Comment:
Keyword is called `params` and not `param`. Please check all examples.

Reply:
done